docs/content/concepts/cryptography/transaction-auth/offline-signing.mdx

```diff
@@ -72,4 +72,20 @@ Amount: Full Balance
 Status : Success
 Mutated Objects:
  - ID: <OBJECT_ID> , Owner: Account Address ( <SUI-ADDRESS> )
+```
+
+## Alternative: Sign with Sui Keystore and Execute Transaction
+
+Alternatively, you can use the active key in Sui Keystore to sign and output a Base64-encoded sender signed data with flag `--serialize-signed-transaction`. 
```

Comment:
```suggestion
Alternatively, you can use the active key in Sui Keystore to sign and output a base64-encoded sender signed data with flag `--serialize-signed-transaction`. 
```
I'm not sure which is more correct, but base64 is used in the rust file comments.